packages/devextreme/js/__internal/grids/grid_core/filter/m_filter_row.ts

```diff
@@ -552,13 +552,15 @@ const ColumnHeadersViewFilterRowExtender = (function () {
         cssClass: `${that.getWidgetContainerClass()} ${CELL_FOCUS_DISABLED_CLASS} ${FILTER_MENU}`,
         showFirstSubmenuMode: 'onHover',
         hideSubmenuOnMouseLeave: true,
-        elementAttr: { 'aria-label': ARIA_SEARCH_BOX },
         items: [{
           disabled: !(column.filterOperations && column.filterOperations.length),
           icon: OPERATION_ICONS[getColumnSelectedFilterOperation(that, column) || 'default'],
           selectable: false,
           items: that._getFilterOperationMenuItems(column),
         }],
+        onItemRendered({ itemElement }) {
+          $(itemElement).attr('aria-label', ARIA_SEARCH_BOX);
```

Comment:
Let's use our method for setting aria attributes
```suggestion
          this.setAria('label', 'ARIA_SEARCH_BOX', itemElement);
```